docs/monitor-kubernetes.md

```diff
@@ -0,0 +1,310 @@
+
+2. This setup is used for a 1:1 mapping from Kubernetes cluster to the PMM server. If you wish to monitor more than one Kubernetes cluster in a single PMM server, you need to configure the dashboard to filter a specific Kubernetes cluster. You also need to properly [relabel the metrics](https://docs.victoriametrics.com/vmagent.html#relabeling) from the backend.
+
+## Pre-requisites
+
+To set up monitoring of Kubernetes, you need the following:
+
+1. PMM Server up and running. You can run PMM Server as a Docker image, a virtual appliance, or on an AWS instance. Please refer to the [official PMM documentation](https://docs.percona.com/percona-monitoring-and-management/setting-up/server/index.html) for the installation instructions.
+
+2. [Helm v3](https://docs.helm.sh/using_helm/#installing-helm).
+
+## Procedure
+
+### Set up authentication in PMM Server
+
+To access the PMM Server resources and perform actions on the server, configure authentication.
+
+1. Get the PMM API key. The key must have the role "Admin".
+
+    === "From PMM UI" 
+
+        [Generate the PMM API key](https://docs.percona.com/percona-monitoring-and-management/details/api.html#api-keys-and-authentication){.md-button} 
+
+    === "From command line"
+
+        You can query your PMM Server installation for the API
+        Key using `curl` and `jq` utilities. Replace `<login>:<password>@<server_host>` placeholders with your real PMM Server login, password, and hostname in the following command:
+
+        ``` {.bash data-prompt="$" }
+        $ API_KEY=$(curl --insecure -X POST -H "Content-Type: application/json" -d '{"name":"operator", "role": "Admin"}' "https://<login>:<password>@<server_host>/graph/api/auth/keys" | jq .key)
+        ```
+
+    !!! note
+
+        The API key is not rotated. 
+
+2. Encode the API key with base64.
+
+    === "in Linux" 
+
+        ````{.bash data-prompt="$" }
+        $ echo -n <API-key> | base64 --wrap=0
+        ````
+
+    === "in macOS"   
+        ```{.bash data-prompt="$" }
+        $ echo -n <API-key> | base64 
+        ```
+
+3. Create the YAML file for the [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) and specify the API key value within. In this example the Secrets file is named `pmm-api-vmoperator.yaml`.
+
+    ```yaml title="pmm-api-vmoperator.yaml"
+    apiVersion: v1
+    data:
+      api_key: <base-64-encoded-API-key>
+    kind: Secret
+    metadata:
+      name: pmm-token-vmoperator
+      #namespace: default
+    type: Opaque
+    ```
+
+4. Create the Namespace where you want to set up monitoring. The following command creates the Namespace `monitoring-system`. You can use another name.
+    
+    ```{.bash data-prompt="$" }
+    $ kubectl create namespace monitoring-system
+    ```
+
+5. Switch the context to the namespace you created. The subsequent commands will be executed in this Namespace.
```

Comment:
No need for step No. 5, because we specify namespace in all steps. Switching context just changes the default namespace to `monitoring-system` to use it instead of `default` if nothing explicitly specified.

Reply:
Updated